### PR TITLE
PW-1288: Feature/3ds2 rebased

### PR DESCRIPTION
**Pull request by Aleffio**
Rebased 3ds2 branch on develop.
Updated credit card logo.
Do not send allow3ds2 on backend orders

### CODE REVIEW

File: app/code/community/Adyen/Payment/Model/Adyen/Abstract.php [hunk 1]
```diff
@@ -681,7 +681,27 @@ protected function processJsonResponse(
 
                 $this->_addStatusHistory($payment, $responseCode, $pspReference, false, $pdfUrl);
                 break;
-            case 'Error':
+            case 'IdentifyShopper':
+                //todo implement result code handling
```
Comment (cyattilakiss) [inline app/code/community/Adyen/Payment/Model/Adyen/Abstract.php]: do you need this todo here?

File: app/code/community/Adyen/Payment/Block/Threeds2.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+class Adyen_Payment_Block_Threeds2 extends Mage_Core_Block_Template
+{
+    public function __construct()
+    {
+        $this->setTemplate('adyen/threeds2.phtml');
+        parent::__construct();
+    }
+
+    protected function _getCheckout()
```
Comment (cyattilakiss) [inline app/code/community/Adyen/Payment/Block/Threeds2.php]: Please don't follow the old habit with the underscored functions :)
Also please add phpdoc where it is possible

File: app/code/community/Adyen/Payment/Model/Adyen/Abstract.php (hunk 1 above)
Comment (Aleffio) [reply app/code/community/Adyen/Payment/Model/Adyen/Abstract.php]: removed

File: app/code/community/Adyen/Payment/Block/Threeds2.php (hunk 2 above)
Comment (Aleffio) [reply app/code/community/Adyen/Payment/Block/Threeds2.php]: removed, where possible

**Pull request merged**